index.js

```diff
@@ -1,21 +1,21 @@
-// Inspired by (but complete rewrite of) requirejs-metagen
-let _ = require('lodash/fp')
+// Inspired by (but complete rewrite of) requirejs-metagen.
```

Comment:
I just generally like writing complete sentences in my comments. Even though it is missing "This is", this one is still actually a complete sentence, grammatically -- I think. And then, the others need period too because one does...! I feel like there is something missing without my trusty periods. But okay, different styles...